### PR TITLE
Loadingpage

### PR DESCRIPTION
**Pull request by seb879**
add a loading message when you start the application

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -60,23 +65,42 @@ class App extends Component {
...
     } = this.props;
     return (
       <div>
-        {barsAreDisplayed && <SearchBar />}
-        { isFirstResearchDone
+        {!poiSampleDisplay.length ? (
+          <div className="loadingPage">
+            <p>
+Attention mesdames et messieurs,
+              <br />
```
Comment (erabin05) [inline src/App.js]: Why ??!!

File: src/App.js [hunk 2]
```diff
@@ -60,23 +65,42 @@ class App extends Component {
...
+          dans un instant,
+              <br />
```
Comment (erabin05) [inline src/App.js]: and again ...

File: src/App.js [hunk 3]
```diff
@@ -60,23 +65,42 @@ class App extends Component {
...
 
-        <AppMap />
-        {Object.keys(specificPoiInfos).length && <PoiInformation />}
-        {!Object.keys(specificPoiInfos).length && barsAreDisplayed && <FilterBar />}
-        {isCreateFormDisplayed && <CreatePoiForm />}
-        {filterKeywordPageDisplay && <SelectCategorie />}
-        {displaySecondImportancePoiPage && <SelectSecondImportancePoi />}
+              <AppMap />
+              {Object.keys(specificPoiInfos).length && <PoiInformation />}
+              {!Object.keys(specificPoiInfos).length && barsAreDisplayed && <FilterBar />}
```
Comment (erabin05) [inline src/App.js:89]: Ternary condition instead for raws 93 and 94

**Review by erabin05 — CHANGES_REQUESTED** [review]
- ```<br/>``` ??
- use ternary condition

File: src/App.js (hunk 1 above)
Comment (seb879) [reply src/App.js]: ok

File: src/App.js (hunk 2 above)
Comment (seb879) [reply src/App.js]: ok

File: src/App.js (hunk 3 above)
Comment (seb879) [reply src/App.js:89]: ok

**Review by erabin05 — APPROVED** [review]
Ok

**Pull request merged**